invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordManagement.js

```diff
@@ -45,7 +45,7 @@ export const RecordManagement = ({
             labelPosition="left"
           >
             <Icon name="edit" />
-            {i18next.t("Edit submission")}
+            {i18next.t("Edit")}
```

Comment:
the name "Edit submission" was done on purpose, to differenciate between curating/editing the community submission, **without creating a new draft**
while "Edit" button is available only after the reocrd is published, and creates new draft. Do you think this differenciation is not needed? ping @lnielsen

Reply:
I think a user won't know the difference if a draft is created or not. For them it's just they want to edit something. I.e. for me it's only the text that's the issue, it's not the behaviour of it.